stdlogr/logger.go

```diff
@@ -17,6 +17,10 @@ func SetVerbosity(v int) {
 	verbosity = v
 }
 
+func GetVerbosity() int {
```

Comment:
Please use `Verbosity()` instead of `GetVerbosity()` (idiomatic approach for Go getters).